### DISCUSSION TITLE
Table Groups in the PBI Service

### DISCUSSION THREAD

**Discussion by sjfinnerty13** [TabularEditor/TabularEditor3#1284] [🙋♀️TE3 Q&A] [answered]
I understand that in Tabular Editor 3, you are able to group tables into a folder structure, but that this folder structure won't pass along to the model in PBI Desktop (According to this video by Haven Consulting https://www.youtube.com/watch?v=3A6eERyo6Rs). 
I'm guessing yes, but wanted to check do this limitation apply to the model in the PBI Service as well? The folder structure created in Tabular Editor 3 can't/won't show up in the service? If so, the feature seems a little pointless beyond just being more organized when modeling in TE3. The ability to have tables grouped in the Power BI model has many applications that would improve its useability and help driver user adoption.

**Comment by otykier** [comment, marked as answer]
It’s a TE3-only feature. I fully agree with your assessment, though, and hope that Microsoft implements a similar feature one day.
